app/models/concerns/audit_extensions.rb

```diff
@@ -31,6 +32,12 @@ def ensure_taxonomies_not_escalated
     end
   end
 
+  def is_auditable_taxable?
+    (auditable.class.included_modules.include?(Taxonomix) ||
+    (auditable.respond_to?(:organization) && auditable.respond_to?(:location))) ||
+    (audited_changes && !audited_changes.include?('location_id') && !audited_changes.include?('organization_id'))
```

Comment:
Both this and line above rely on fact that resource that does not include Taxonomix always has organization and location supported. That is fact for Host only and also in case that both are enabled via settings. However, that's not the case for most of Katello resrouces, they scope only by organization. I believe both `&&` between location and organization should in fact be `||`. Set taxonomies is luckily smart enough to only set the taxonomy it has seen.